src/components/ContactList/ZeroContacts.js

```diff
@@ -19,5 +19,22 @@ export default function ZeroContacts() {
 
 const ListWrapper = styled.div`
   display: grid;
+  width: 100%;
+  justify-items: center;
+  margin: 0 auto;
   margin-top: 50px;
 `
+
+const NotificationStyled = styled.p`
+  color: white;
+  text-align: center;
+`
+
+const ButtonStyled = styled.button`
+  padding: 17px 48px;
+  font-size: 20px;
+  background: #ffffff 0% 0% no-repeat padding-box;
```

Comment:
I would define your colors in your GlobalStyles, especcially if you are using them multiple times